src/main/blackbox/blackbox.c

```diff
@@ -1119,6 +1133,7 @@ static void loadMainState(timeUs_t currentTimeUs)
         blackboxCurrent->axisPID_D[i] = lrintf(pidData[i].D);
         blackboxCurrent->axisPID_F[i] = lrintf(pidData[i].F);
         blackboxCurrent->gyroADC[i] = lrintf(gyro.gyroADCf[i] * blackboxHighResolutionScale);
+        blackboxCurrent->gyroUF[i] = lrintf(gyro.gyroADC[i] * blackboxHighResolutionScale);
```

Comment:
@tbolin please check
```suggestion
        blackboxCurrent->gyroUF[i] = lrintf(gyro.gyroADCf[i] * blackboxHighResolutionScale);
```